docs/pages/eas-workflows/jobs.mdx

```diff
@@ -3,105 +3,137 @@ title: Jobs
 description: Learn which pre-packaged and custom jobs you can run with EAS Workflows.
 ---
 
-EAS Workflows sequence jobs together to automate your development and release processes. You can run two types of jobs: pre-packaged jobs and custom jobs.
+EAS Workflows sequence jobs together to automate your development and release processes. You can run two types of jobs: pre-packaged jobs and custom jobs. Pre-packaged jobs use a `type` property to specify the job type. Custom jobs run custom code that you define in the workflow configuration.
 
-Expo provides pre-packaged jobs that meet common use cases for developers, such as building, submitting, and running end-to-end tests. Since they are provided by Expo, they are continuously maintained and updated.
+## Job properties
 
-Custom jobs are jobs that you write yourself. They are useful for any use case not met by the pre-packaged jobs.
+All jobs support the following properties:
 
-## Build
+- `name` (optional): The name of the job, displayed on the Expo dashboard.
+- `if` (optional): Conditional expression that determines if the job should run. Limited to 250 characters.
+- `needs` (optional): List of job keys that must complete successfully before this job runs
+- `after` (optional): List of job keys that must complete (success or failure) before this job runs
 
-This is a pre-packaged job that creates a build from a project:
+## Build your app
+
+Creates a build from your project:
 
 ```yaml .eas/workflows/build.yaml
-jobs:
-  build:
-    type: build
-    params:
-      platform: android | ios
-      profile: string # default: production
+build:
```

Comment:
Hmm, I'm not sure it's a good decision to skip the `jobs:`, because without it, the file suddenly looks very much like a Custom Build YAML which is doubly confusing.